Model/Observer/AdminConfigs/CopyConfigurationOptionUpdated.php

```diff
@@ -0,0 +1,136 @@
+            StoreSearchConfig::WIZZY_SEARCH_ENDPOINT,
+            StoreSearchConfig::WIZZY_NO_OF_PRODUCTS,
+            StoreSearchConfig::WIZZY_DISPLAY_ADD_TO_CART_BUTTON,
+            StoreSearchConfig::WIZZY_DISPLAY_ADD_TO_WISHLIST_BUTTON,
+            StoreSearchConfig::WIZZY_FACETS,
+            StoreSearchConfig::WIZZY_FACET_CATEGORY_DISPLAY,
+            StoreSearchConfig::WIZZY_LEFT_FACETS_COLLAPSIBLE,
+            StoreSearchConfig::WIZZY_LEFT_FACETS_DEFAULT_COLLAPSIBLE_BEHAVIOUR,
+            StoreSearchConfig::WIZZY_LEFT_FIRST_FACET_DEFAULT_COLLAPSIBLE_BEHAVIOUR,
+            StoreSearchConfig::WIZZY_PAGINATION_TYPE,
+            StoreSearchConfig::WIZZY_PAGINATION_MOVE_TO_TOP_WIDGET,
+            StoreSearchConfig::INFINITE_SCROLL_OFFSET_DESKTOP,
+            StoreSearchConfig::INFINITE_SCROLL_OFFSET_MOBILE,
+            StoreSearchConfig::WIZZY_SWATCHES_CONFIGURATION,
+            StoreSearchConfig::WIZZY_SORT_CONFIGURATION,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_SUGGESTIONS_COUNT,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_CATEGOIRES_TITLE,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_OTHERS_TITLE,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_BRANDS_TITLE,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_SECTIONS,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_MENU_ALIGNMENT,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_NO_RESULTS_BEHAVIOUR,
+            StoreAutocompleteConfig::WIZZY_AUTOCOMPLETE_NO_RESULTS_TEXT,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_SHOW_TOP_PRODUCTS,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_TOP_PRODUCTS_TITLE,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_TOP_PRODUCTS_COUNT,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_HAS_TO_IGNORE_CATEGORIES,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_CATEGORIES_TO_IGNORE,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_PAGES_TITLE,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_EXCLUDE_PAGES,
+            StoreAutocompleteConfig::WIZZY_AUTTOCOMPLETE_SYNC_PAGES,
+            StoreAutocompleteConfig::AUTOCOMPLETE_ENABLED_ATTRIBUTES,
+            StoreAdvancedConfig::IS_OVERRIDING_EVENTJS,
+            StoreAdvancedConfig::INCLUDE_CUSTOM_CSS,
+            StoreAdvancedConfig::TEMPLATE_ATTRIBUTES
+        ];
+
+        foreach ($configPath as $path) {
+            $oldConfig[$path] = $this->configManager->getStoreConfig($path, $fromStoreId);
+        }
+
+        foreach ($configPath as $path) {
+            $newConfig[$path] = $this->configManager->getStoreConfig($path, $toStoreId);
+        }
+        foreach ($oldConfig as $path => $value) {
+            $this->_configWriter->save(
+                $path,
+                $value,
+                $scopeType,
+                $toStoreId
+            );
+        }
+
+        $cacheType = 'config';
```

Comment:
Do we have any const available? If yes, use it please.

Reply:
I think there is no any Const for cacheType , we have to pass config type as string only.